src/GovUk.Education.ExploreEducationStatistics.Data.Model.Tests/Fixtures/FilterGroupGeneratorExtensions.cs

```diff
@@ -17,6 +17,11 @@ public static Generator<FilterGroup> WithDefaults(this Generator<FilterGroup> ge
     public static Generator<FilterGroup> WithFilter(this Generator<FilterGroup> generator, Filter filter)
         => generator.ForInstance(s => s.SetFilter(filter));
 
+    public static Generator<FilterGroup> WithFilterItems(
```

Comment:
This method is actually here so that we can do stuff like this in consuming code:

```cs
WithFilterItems(filterItems[..2]); // multiple
WithFilterItems(filterItems[0]); // singular 
```

Main benefit is that we don't need to convert them to lists. Tbh, it's not really a big deal to convert them so have just removed this extension now.